dev/Pager/PrototypePager/Themes/PrototypePager.xaml

```diff
@@ -0,0 +1,181 @@
+                                        FontFamily="Segoe MDL2 Assets"
+                                        Content="&#xE892;"/>
+                                <TextBlock x:Name="FirstPageButtonText" VerticalAlignment="Center" HorizontalAlignment="Center"
+                                           Text="{TemplateBinding FirstPageButtonText}"
+                                           Visibility="{Binding ElementName=FirstPageButton, Path=Visibility}"/>
+                                <Button x:Name="PreviousPageButton"
+                                        IsEnabled="False"
+                                        Style="{StaticResource NavigationBackButtonNormalStyle}"
+                                        FontFamily="Segoe MDL2 Assets"
+                                        Content="&#xE76B;"/>
+                                <TextBlock x:Name="PreviousPageButtonText" VerticalAlignment="Center" HorizontalAlignment="Center"
+                                           Text="{TemplateBinding PreviousPageButtonText}"
+                                           Visibility="{Binding ElementName=PreviousPageButton, Path=Visibility}"/>
+                            </StackPanel>
+                            <StackPanel x:Name="BoxPanels" Visibility="Collapsed" Orientation="Horizontal" Grid.Row="0" Grid.Column="1">
+                                    <TextBlock VerticalAlignment="Center" Margin="0,0,5,0" Text="{TemplateBinding PrefixText}" />
+                                    <!--NumberBox Display-->
+                                    <controls:NumberBox x:Name="NumberBoxDisplay"
+                                                    HorizontalAlignment="Center"
+                                                    VerticalAlignment="Center"
+                                                    Maximum="{Binding RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource IntToDouble}, Path=NumberOfPages}"
+                                                    Minimum="1"
+                                                    Value="{Binding RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource IntToDouble}, Path=SelectedIndex, Mode=TwoWay}"
+                                                    SmallChange="1"
+                                                    LargeChange="5"
+                                                    IsEnabled="False"
+                                                    Visibility="Collapsed" />
+                                    <!--ComboBox Display-->
+                                    <ComboBox x:Name="ComboBoxDisplay"
+                                          HorizontalAlignment="Center" VerticalAlignment="Center"
+                                          SelectedIndex="{Binding RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource IntMinusOne}, Path=SelectedIndex, Mode=TwoWay}"
```

Comment:
>Converter={StaticResource IntMinusOne} [](start = 115, length = 38)

@ranjeshj and @mikehillberg I'm not sure how I feel about this... I think by presendence this would be a template setting, not a converter.